holmes/core/tools.py

```diff
@@ -111,6 +111,9 @@ def get_openai_format(self):
 
         return result
 
+    def log(self, params: Dict):
+        logging.info(f"Running tool: {self.__class__.__name__} with params {params}")
```

Comment:
I think we should at least log the name of the tool which is often different from the class name.
I will care for updating this PR. The plan is to have something similar to:

```python
class Tool(ABC):
    def invoke(self, params):
        logging.info(f"Running tool: {self.get_parameterized_one_liner(context)}")
        return self._invoke(params)

    @abstractmethod
    def _invoke(self, params):
        pass
```

Reply:
Sounds good, thanks